### DISCUSSION TITLE
Would it be possible to release version 9.4?

### DISCUSSION THREAD

**Discussion by lesshaste** [google/or-tools#3378] [General] [answered]
Looking at the milestones the only remaining issues seem to be packaging related. It would be really great if 9.4 could be released (please).

**Comment by lperron** [comment]
We try. Linux on arm64 is problematic. And we cannot add it after the fact with java and .NET.

**Reply by lperron** [reply]
You can build from source. It is quite easy using cmake

**Comment by lperron** [comment, marked as answer]
Hopefully next week.

**Comment by Mizux** [comment]
Please follow:
* #3341 

ps: I will try to keep up to date with all issues blocking the release artifact(s) generation
